skytemple_files/graphics/chara_wan/dbg/test.py

```diff
@@ -0,0 +1,63 @@
+import sys
+import os
+import shutil
+import math
+import glob
+import chara_wan.writer as exWriter
+
+
+
+def getWanFilePath(baseDir, dirName, index):
```

Comment:
Can you please use `snake_case` for all functions, variables and methods? If you want I can probably auto-convert it.